packages/core/src/Modules/ComplaintsPolicy/Components/complaints-policy-content.jsx

```diff
@@ -171,7 +190,36 @@ const getYourDecisionText = landing_company_shortcode => {
     }
 };
 
-const Content = ({ landing_company_shortcode, mt5_login_list }) => {
+const getSubmissionOfAComplaintText = landing_company_shortcode => {
+    let text =
+        'To file a complaint about our service, send an email to <0>complaints@deriv.com</0> and state your complaint in detail. Please submit any relevant screenshots of your trading or system for our better understanding.';
+    const components = [
+        <a
+            key={0}
+            className='link link--orange'
+            rel='noopener noreferrer'
+            target='_blank'
+            href='mailto:complaints@deriv.com'
+        />,
+    ];
+
+    if (landing_company_shortcode !== 'maltainvest') {
+        text += ' You may also call <1>+447723580049</1> to place your complaint.';
+        components.push(
+            <a
+                key={1}
+                className='link link--orange'
+                rel='noopener noreferrer'
+                target='_blank'
+                href='tel:+447723580049'
+            />
+        );
+    }
+
+    return <Localize i18n_default_text={text} components={components} />;
```

Comment:
Can you confirm if `Localize` works properly if we pass `{text}` in this manner?

Reply:
yes, this will not work, we have to pass the exact string to the i18n_default_text props @AbdallaMourad

Reply:
```suggestion
    return <Localize i18n_default_text="To file a complaint about our service, send an email to <0>complaints@deriv.com</0> and state your complaint in detail. Please submit any relevant screenshots of your trading or system for our better understanding." components={components} />;
```

Reply:
The working solution is to have separate `Localize` string wrapper components with pre-defined `text` for each case

Reply:
what you can do is to have two Localize components inside of a inline element to add the new text, and add logic to render them